contrib/cirrus/check_image.sh

```diff
@@ -36,4 +36,28 @@ do
         "$(systemctl list-unit-files --no-legend $REQ_UNIT)" = "$REQ_UNIT enabled" || let "RET+=1"
 done
 
+# Previously, perdiodic package updates were enabled and would trigger
+# soon after an Ubuntu VM booted, grabbing a specific lock file for
+# several minutes after setup_environment.sh ran (previous script here also).
+DPKGLCK=/var/lib/dpkg/lock
+if [[ "$OS_RELEASE_ID" == "ubuntu" ]] && [[ -r "$DPKGLCK" ]]
+then
+    item_test "The lock file $DPKGLCK exists" -r "$DPKGLCK" || let "RET+=1"
```

Comment:
I don't understand this. What is the purpose of testing immediately after having done so in line 43? Did you intend to remove the `[[ -r` on line 43 and lines 56-59?

Reply:
Oops, yes that doesn't make much sense, probably an artifact of an older implementation.